draft-ietf-rats-architecture.md

```diff
@@ -127,6 +127,79 @@ Appraisal Policy to make application-specific decisions such as authorization de
 The Attestation Result Appraisal Policy might, for example, be configured in the Relying Party 
 by an administrator.
 
+## Composite Attester {#compositeattester}
+
+A Composite Attester is an entity composed of multiple sub-entities that its
+trustworthiness has to be determined by evaluating all these sub-entities.
+Each sub-entity has at least one Attesting Environment collecting the claims from
+at least one Target Environment and generates its Evidence against its trustworthiness,
```

Comment:
since "collecting" is in present tense, should, "generates" -> "generating"?

Reply:
I intend to use 'Each sub-entity' as the subject for the verb 'generates'

Reply:
Text is changed to this:
Each sub-entity has at least one Attesting Environment collecting the claims from at least one Target Environment, then this sub-entity generates Evidence about its trustworthiness. Therefore each sub-entity can be called an Attester.

Reply:
I am fine with entities and sub-entities. Maybe we will have to map these terms to specific entity instances, such as a composite device and components. At least we have to lay out somewhere in this document that the relationship between each tuple of terms is the same. Also, as they are our "bucket for roles" I think entities and sub-entities should be added to the terminology section as highlighted in issue https://github.com/ietf-rats-wg/architecture/issues/19.